client/components/Item/Card/Card.vue

```diff
@@ -69,22 +66,29 @@
           </div>
         </div>
       </div>
-      <div v-if="text" class="card-text">
-        <div class="card-title mt-1 text-truncate">{{ cardTitle }}</div>
-        <nuxt-link
-          v-if="item.Type === 'MusicAlbum' && item.AlbumArtists.length > 0"
-          tag="div"
-          class="card-subtitle text--secondary text-truncate link"
-          :to="getItemDetailsLink(item.AlbumArtists[0], 'MusicArtist')"
-        >
-          {{ cardSubtitle }}
-        </nuxt-link>
-        <div v-else class="card-subtitle text--secondary text-truncate">
-          {{ cardSubtitle }}
-        </div>
+    </component>
+    <!-- CAPTIONS -->
```

Comment:
I don't think these title comments help anything personally.

Reply:
If there's the need to add comments to a template, it's time to split that part of the template into a component, imo.

Reply:
Yeah no worry, it was there to help me visualise where the stuff was, but it's not needed anymore